projects/js-packages/social-logos/src/react/social-logo-data.jsx

```diff
@@ -0,0 +1,231 @@
+/** This is a generated file. Do not edit. */
```

Comment:
I still wish we didn't have to have this committed. I wonder if it'd work to have a stub version here that loads the built version if available and otherwise slurps in the svgs from `src/svg/` at runtime. They won't be quite the same svgs since they won't have been processed by `tools/svg-optimize`, but will they work?

Or would that be too much trouble compared to writing a test to make sure this has been rebuilt properly?

Reply:
I think the answer is "yes, that's too much trouble". We'll need that test then.

Reply:
Technically I could put a stub with one or two icons (or no icons), and the component handles missing icons gracefully with a blank SVG. But that's not helpful for the actual dev experience, as we want to be able to see the correct icons during development.

As for slurping the SVGs, it's a bit more complicated there too, as we'd need to convert the SVG attributes to appropriate camelCase like the current 1→2 process does.

As for a CI test (which is what I believe you're referring to), the easiest thing would be to run 1→2 and make sure the file didn't change, right?

Reply:
> As for slurping the SVGs, it's a bit more complicated there too, as we'd need to convert the SVG attributes to appropriate camelCase like the current 1→2 process does.

Also we'd need to convert them from strings into React components. That was the point where I gave up, that's possible but it's getting really complicated on top of the big difference it makes in the webpack bundle to be slurping in the files.

> As for a CI test (which is what I believe you're referring to), the easiest thing would be to run the full build and make sure the file didn't change, right?

That'll work, yeah.